typing/typemod.ml

```diff
@@ -2551,12 +2551,6 @@ and type_structure ?(toplevel = None) funct_body anchor env sstr =
         let () = if rec_flag = Recursive then
           Typecore.check_recursive_bindings env defs
         in
-        if Option.is_some toplevel then begin
-          (* Values bound by '_' still escape in the toplevel, because
-              they may be printed even though they are not named *)
-          defs |> List.iter (fun vb ->
-            Typecore.escape ~loc:vb.vb_pat.pat_loc ~env:newenv vb.vb_expr.exp_mode);
-        end;
```

Comment:
this hack is removed for simplicity; causing one test in `local.ml` failing; other workaround in https://github.com/ocaml-flambda/ocaml-jst/pull/2

Reply:
I don't understand. Why is this OK to remove?

Reply:
Sorry for the confusion - it's not OK to remove, and I will ask @stedolan how to restore it somewhere else.

Reply:
This issue is fixed by https://github.com/ocaml-flambda/ocaml-jst/pull/104